.changeset/violet-lamps-relate.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/ui-extensions-react': patch
```

Comment:
this should be a minor, not a patch as it adds new features
https://github.com/Shopify/ui-extensions/blob/unstable/documentation/versions-and-deploys.md#adding-code

Reply:
Makes sense. I've seen component releases such [as Switch](https://github.com/Shopify/ui-extensions/pull/2014/files) being a patch, though.

@oliverigor Just double checking with you if we should set this as a `minor` release instead.

Reply:
Yeah, it has been a back and fourth with this, but I believe Ryan is correct, should be a minor as this is a release of a new component. 

Sorry for steering you in the wrong direction initially :( 
I should create clear guidelines for us on when to do what regarding the changesets.

Reply:
Parfait! Updated the patch to `minor` :D